utilities/openemr-monitor/grafana/provisioning/datasources/datasource.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: 1
+
+datasources:
+  - name: prometheus
+    type: prometheus
+    access: proxy
+    orgId: 1
+    url: http://hostIp:9090
```

Comment:
would it make sense to default this to `localhost` rather than `hostIp` ?

Reply:
@bradymiller sure